Jason/book_class.py

```diff
@@ -0,0 +1,74 @@
+
+# my_library
+class Booklist():
+
+# Set up the initialization function for class Booklist()
+# The function should initialize just 1 attribute, called 'books', as an empty list
+# No parameters needed, other than self     
+
+	def __init__(self):
+		self.books = []
+		
+# Define the add() method to add a book to an object of class Booklist
+# -The method should take 2 parameters other than self, both strings -- 'title', and 'author'
+# -The method should make a book variable as a dictionary with two key/value pairs for title/author
+# -Then, the method should append this book to books attribute of the the Booklist object  - i.e. self.books.append(book)
+
+	def add(self, title, author):
+		book = {'title' : title , 'author' : author}
```

Comment:
Great way to reduce amount of code needed. Rather than setting an empty dictionary and setting parameters to dictionary keys separately, you were able to get that completed in one line